backend/src/main/ast/Program.java

```diff
@@ -1,6 +1,6 @@
-package ast;
+package main.ast;
```

Comment:
this line in all frontend files

Reply:
What about this line

Reply:
I was not able to run anything without this line

Reply:
This line depends on your project structure settings. All my files will now have conflicts because the package name is different. Not to mention, you changed the name frontend to backend, which causes even more conflicts. I understand your opinion about it and respect it(though the compiler construction class teaches otherwise) but we should have dealt with that before you merged the initial-setup branch to master. Now, I have a different folder name and files. That is the conflict I am talking about.

Reply:
Ok, like I said, project structure settings can be changed easily. I also said I am willing to handle all the merge conflicts. You can continue doing everything you have been doing, and merge that to master with no issues. My code is still on a separate branch and I can merge master before merging to master and fix all conflicts. 
You've still not explained why you insist on calling this front end code, when there is nothing being output to a UI of any type. Google sheets API handles all front end rendering of all the data our application will handle. Regardless, a filename change is very simple to handle merge conflicts with, which again, I've said I would handle. 
If we can all agree on a project structure that everyone can use, then handling conflicts that will arise is not an issue.